docs/GettingStarted/DockerComposeSetup.md

```diff
@@ -34,11 +34,11 @@ Support for database schema migration was introduced to DevLake in v0.10.0. From
 <br/>
 
 ## FAQ
-### Can I use an external Database service instead of running Database in Docker?
+### Can I use a managed Cloud database service instead of running database in Docker??
 
-    Yes, you can simply comment out the 'mysql' part in 'docker-compose.yml' and update some configurations in '.env' before you run docker compose up -d, here are the steps:
+Yes, you can simply comment out the 'mysql' part in 'docker-compose.yml' and update some configurations in '.env' before you run docker compose up -d, here are the steps:
 
-1. Comment out mysql part:
+1. Comment out 'mysql' part in `docker-compose.yml`:
```

Comment:
"Comment out 'mysql' part..." => "Comment out the 'mysql' section..."

Reply:
fixed